src/test/java/cloud/fogbow/ras/core/plugins/interoperability/aws/image/v2/AwsV2ImagePluginTest.java

```diff
@@ -61,75 +62,105 @@ public void setUp() {
         this.plugin = Mockito.spy(new AwsV2ImagePlugin(awsConfFilePath));
     }
 
-    // test case: check if getAllImages returns all images expected in the expected format.
+    // test case: check if getAllImages returns all images expected in the expected
+    // format and if the right calls are made.
     @Test
     public void testGetAllImages() throws FogbowException {
         // setup
         Ec2Client client = Mockito.mock(Ec2Client.class);
         PowerMockito.mockStatic(AwsV2ClientUtil.class);
         BDDMockito.given(AwsV2ClientUtil.createEc2Client(Mockito.anyString(), Mockito.anyString())).willReturn(client);
 
-        List<software.amazon.awssdk.services.ec2.model.Image> imagesList = getMockedImages();
+        DescribeImagesResponse response = DescribeImagesResponse.builder().images(getMockedImages()).build();
+        PowerMockito.mockStatic(AwsV2CloudUtil.class);
+        BDDMockito.given(AwsV2CloudUtil.doDescribeImagesRequest(Mockito.any(), Mockito.any())).willReturn(response);
+
+        List<Image> imagesList = getMockedImages();
 
         AwsV2User cloudUser = Mockito.mock(AwsV2User.class);
-        DescribeImagesRequest imagesRequest = DescribeImagesRequest.builder().owners(cloudUser.getId()).build();
 
         List<ImageSummary> expectedResult = new ArrayList<>();
-        for (Image each: imagesList) {
+        for (Image each : imagesList) {
             expectedResult.add(new ImageSummary(each.imageId(), each.name()));
         }
 
-        Mockito.when(client.describeImages(imagesRequest)).thenReturn(DescribeImagesResponse.builder().images(imagesList).build());
-
         // exercise
-       List<ImageSummary> result = this.plugin.getAllImages(cloudUser);
+        List<ImageSummary> result = this.plugin.getAllImages(cloudUser);
 
-        //verify
+        // verify
         Assert.assertEquals(expectedResult, result);
+        PowerMockito.verifyStatic(AwsV2ClientUtil.class, Mockito.times(1));
+        AwsV2ClientUtil.createEc2Client(Mockito.anyString(), Mockito.anyString());
+
+        PowerMockito.verifyStatic(AwsV2CloudUtil.class, Mockito.times(1));
+        AwsV2CloudUtil.doDescribeImagesRequest(Mockito.any(), Mockito.any());
+
+        Mockito.verify(plugin, Mockito.times(1)).mountImagesSummary(Mockito.any());
     }
 
-    // test case: check if the getImage returns the correct image when there are some.
+    // test case: check if the getImage returns the correct image when there are some and if the right calls are made.
     @Test
     public void testGetImageWithResult() throws FogbowException {
         //setup
         Ec2Client client = Mockito.mock(Ec2Client.class);
         PowerMockito.mockStatic(AwsV2ClientUtil.class);
         BDDMockito.given(AwsV2ClientUtil.createEc2Client(Mockito.anyString(), Mockito.anyString())).willReturn(client);
 
-        List<software.amazon.awssdk.services.ec2.model.Image> imagesList = getMockedImages();
+        List<Image> imagesList = getMockedImages();
```

Comment:
- Remove the unused variable declaration.
- There is also unused import in line 31, remove it too.